test/dummy/app/views/messages/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= content_for :head do %>
+  <script type="module">
+    import Rails from "https://cdn.skypack.dev/@rails/ujs"
+
+    const { delegate, disableElement, enableElement } = Rails
```

Comment:
Similar to the question raised on hotwired/turbo#40 (https://github.com/hotwired/turbo/pull/40#discussion_r548945358), it isn't clear on where this code should live.

One option is to declare it as an engine-provided HTML partial declaring the `<script type="module">` in the same style as this test currently does. Consumers could call it like any other Turbo-provided helpers. Unfortunately, there wouldn't be a seam for them to provide their applications already loaded `Rails` from `@rails/ujs`.

Another alternative would be to declare a separate `@turbo-rails/ujs` module for consumers to import and load (calling something like `startUJS(Rails)` so that they can provide their own already started `Rails`. Unfortunately, during my initial attempts at that, it wasn't trivial to import _that_ module into this engine's test suite's `test/dummy/packs/application.js`. 

If there's a clear path forward, I'll re-structure this block in any way that suits us. In its current form, it at least provides confidence that a polyfill is possible and able to be covered by automated tests.

Reply:
Once we decide on how consumers can consume it, I'll follow up with some documentation changes (to the README or elsewhere).